playground-3.js

```diff
@@ -1,3 +1,24 @@
 // https://miorita.scoalaweb.ro/playgrounds/escape
 // Scrie un program care să o ajute pe Miorița să ajungă la pășune
 // Programul trebuie să funcționeze indiferent de configurația stânei.
+function turnAround(){
+    turnRight();
+}
+
+while(! canMove()){
+    turnAround();
+}
+
+function superMove(){
+    let n=[];
+    for(let i=n.length;i>=0;i++){
```

Comment:
`for`-ul tău va rula la infinit.

Mărimea unui array (`n.length`) nu poate fi niciodată mai mică decât 0.   
Tu pornești cu `i` de la lungimea listei, apoi îl tot crești.

Reply:
Ok, am refacut codul. 
Ideea mea initiala a fost sa ma gandesc la un sir cu n caractere si sa porneasca cum ar veni de la sfarsitului sirului si sa se opreasca cand ajunge la index 0. De asta ma gandeam initial ceva de genu (i=n.length; i>=0 ; i--). Doar ca nu imi reuseste asa.